### DISCUSSION TITLE
Duda ejercicio 6 TP2: Suma de Números Primos

### DISCUSSION THREAD

**Discussion by damianperezz** [INGCOM-UNRN-P1/cursada-2023#74] [Preguntas y respuestas] [answered]
![image](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/141683317/e16888a0-fa54-423e-9483-0b88500982e6)

lo que pide este codigo es ingresar un numero y sumar los numeros primos comprendidos desde el 0 hasta ese numero?
o lo que esta pidiendo es sumar cierta cantidad de numeros primos, indicado por el numero ingresado?

**Comment by martinvilu** [comment, marked as answer]
El `n` es la cantidad de números primos.

Usando https://oeis.org/A000040 como lista de números primos y `n` en `10`, el resultado esperado sería algo como 129.

(Los primeros 10 números primos serían {2, 3, 5, 7, 11, 13, 17, 19, 23, 29})
